### PR TITLE
Additions in Game List

### PR DESCRIPTION
**Pull request by TheExentist151**
Added a notice that Sonic Colors uses NintendoWare.
Added Mario & Sonic at the Sochi 2014 Olympic Winter Games.

### CODE REVIEW

File: docs/docs/hedgehog-engine/gamelist.md [hunk 1]
```diff
@@ -5,16 +5,21 @@ description: A list of games that are using Hedgehog Engine.
...
-| Sonic Generations                       | BlueBlur      |
-| Sonic Lost World                        | sonic2013     |
-| Sonic Colors: Ultimate\*                | Rainbow       |
+| Game Name                                              | Internal Name |
+| ------------------------------------------------------ | ------------- |
+| Sonic Unleashed (Sonic World Adventure)                | SWA           |
+| Sonic Colors\*                                         | sonic2010     |
+| Sonic Generations                                      | BlueBlur      |
+| Sonic Lost World                                       | sonic2013     |
+| Mario & Sonic at the Sochi 2014 Olympic Winter Games\* | urals         |
```
Comment (RagdollClash) [inline docs/docs/hedgehog-engine/gamelist.md]: We looked into Sochi 2014 before, the only thing it has in common with Hedgehog Engine is it using the lightfield code.

**Review by RagdollClash — CHANGES_REQUESTED** [review]
Sochi 2014 shouldn't be added to this list.

File: docs/docs/hedgehog-engine/gamelist.md (hunk 1 above)
Comment (TheExentist151) [reply docs/docs/hedgehog-engine/gamelist.md]: okay.

**Comment by PTKay** [comment]
This PR doesn't seem necessary to be honest:

- Sochi doesn't use the Hedgehog Engine;
- NintendoWare isn't an engine, but a middleware, and Colours only uses some file formats of said middleware. This doesn't seem relevant to be listed in the game list. The only notes we add is if the game it using a different engine together with HE somehow, which is the case of SCU, but not the original Colours.

Due to the reasons stated above, I'm closing this PR.

**Pull request closed without merging**